etc/ATSCam/primary_hdu.header

```diff
@@ -10,15 +10,15 @@ DATE-OBS=                      / Date of the observation (image acquisition)
 DATE-END=                      / end date of the observation
 MJD     =                      / Modified Julian Date that the file was written
 MJD-OBS =                      / Modified Julian Date of observation
-OBS-NITE=                      / Night of the observation
-OBS-ID  =                      / ImageName from Camera StartIntergration
-IMGTYPE =                      / BIAS, DARK, FLAT, OBJECT
+OBSID   =                      / ImageName from Camera StartIntergration
+GROUPID =                      / imageSequenceName from Camera StartIntergration
+OBSTYPE =                      / BIAS, DARK, FLAT, OBJECT
 BUNIT   = 'adu     '           / Brightness units for pixel array
 
          ---- Telescope info, location, observer ----
 TELESCOP= 'LSST AuxTelescope'  / Telescope name
 INSTRUME= 'LATISS'             / Instrument used to obtain these data
-OBSERVER=                      / Observer name(s)
+OBSERVER= 'LSST'               / Observer name(s)
```

Comment:
I don't understand why this is present.

Reply:
This is according to the definition in:
https://confluence.lsstcorp.org/pages/viewpage.action?spaceKey=SYSENG&title=Auxiliary+Telescope+Header+Information+Topic+Mapping

Reply:
Ok, It's not in LSE-400 draft and it makes no sense being there if we have no current means for envisaging how the names of the multiple people on the mountain will end up in the file header and it's not something which matters for a survey telescope (it can be found by other means).